### PR TITLE
Own node

### PR DESCRIPTION
**Pull request by Parsh**

### CODE REVIEW

**Comment by AliMeer** [comment]
@parsh Lets not merge this yet. We can merge after 1.4.0

File: src/pages/ManageBackup/index.tsx [hunk 1]
```diff
@@ -803,6 +804,21 @@ export default function ManageBackup(props) {
...
+//     }
+//     const stringified =  JSON.stringify(ownNode)
+//     console.log({stringified})
+//     await AsyncStorage.setItem("OwnNodes", stringified)
+
+//   config.connectToOwnNode();
+//  })()
+    
+//   }, [])
+
```
Comment (CypherPoet) [inline src/pages/ManageBackup/index.tsx:821]: Would it be better to delete this?

Comment (Parsh) [reply src/pages/ManageBackup/index.tsx:821]: Yea, we'll shift this to the own node flow once it's in place.

File: src/bitcoin/HexaConfig.ts [hunk 2]
```diff
@@ -269,6 +272,31 @@ class HexaConfig {
...
+  public connectToOwnNode =  async () => {
+    let ownNodes:any = await AsyncStorage.getItem('OwnNodes')
+
+    if(ownNodes){
+      ownNodes = JSON.parse(ownNodes);
+      const ownNodeURL = ownNodes.urls[ownNodes.active];
+      if(ownNodeURL){
+        this.ESPLORA_API_ENDPOINTS = {
+          ...this.ESPLORA_API_ENDPOINTS,
+           MAINNET: {
```
Comment (AliMeer) [inline src/bitcoin/HexaConfig.ts:285]: @Parsh Is the assumption here that own node will only work for MAINNET and not on TESTNET?
The testnet account can continue to use Hexa Node/wrapper. However for dev env will we be able to test own node functionality and if so will MAINNET object have testnet end points in that case?

Comment (Parsh) [reply src/bitcoin/HexaConfig.ts:285]: Yes, I don't think anyone is going to spin up a Testnet node w/ a Testnet indexer just to make their Testnet account work. Also, the standby indexer that Varun has is mainnet one, so we'll be testing this on staging, hence we ignored the testnet configuration altogether.

**Review by AliMeer — APPROVED** [review]

**Pull request merged**